plugin/geoip/setup_test.go

```diff
@@ -0,0 +1,114 @@
+package geoip
+
+import (
+	"fmt"
+	"net"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+
+	"github.com/coredns/caddy"
+	"github.com/coredns/coredns/core/dnsserver"
+)
+
+var (
+	_, file, _, _    = runtime.Caller(0)
+	fixturesDir      = filepath.Join(filepath.Dir(file), "testdata")
+	enterpriseDBPath = filepath.Join(fixturesDir, "GeoIP2-Enterprise.mmdb")
+	cityDBPath       = filepath.Join(fixturesDir, "GeoLite2-City.mmdb")
+	countryDBPath    = filepath.Join(fixturesDir, "GeoLite2-Country.mmdb")
```

Comment:
I think we're only using `city`? So we don't need to test anything else here.

Reply:
You are right, we are using `city` schema only, however multiple database files provides that schema, `GeoIP2-Enterprise.mmdb`, `GeoLite2-City.mmdb` and `GeoLite2-Country.mmdb` all provide the city schema, Details can be found in the code of the geoip2-golang package, [see](https://github.com/oschwald/geoip2-golang/blob/main/reader.go#L273-L310).

The other database types are used to test unknown, or unsupported database types.

Does this make sense to you?